src/main/groovy/com/pszymczyk/consul/infrastructure/ConsulWaiter.groovy

```diff
@@ -15,14 +15,22 @@ class ConsulWaiter {
     private final int port
 
     ConsulWaiter(String host, int port) {
```

Comment:
4 constructors with mix of raw parameters do not looks good, are they already used somewhere?

Reply:
`ConsulWaiter(String host, int port)` seems to be unused, I will remove it